presto-record-decoder/pom.xml

```diff
@@ -56,6 +56,11 @@
             <artifactId>avro</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.confluent</groupId>
```

Comment:
Is schema-registry only supported with confluent? Is there any community version of schema registry that comes with apache kafka?

Reply:
There are other types of schema registries but they have different api's and are not distributed with apache kafka , this one is only supported with confluent, it's the most widely used one.